wsit/boms/bom/pom.xml

```diff
@@ -68,6 +68,24 @@
                 <scope>import</scope>
             </dependency>
 
+            <dependency>
+                <groupId>com.sun.xml.ws</groupId>
+                <artifactId>jaxws-rt</artifactId>
+                <version>${jaxws.ri.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>com.sun.xml.ws</groupId>
+                <artifactId>policy</artifactId>
+                <version>${jaxws.ri.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>com.sun.xml.bind</groupId>
+                <artifactId>jaxb-impl</artifactId>
+                <version>${jaxws.ri.version}</version>
+            </dependency>
+
```

Comment:
remove all these deps (jaxws-rt, policy, jaxb-impl) they're already available through the dependency with import scope above. Other than that, policy as such can be removed completely from all places since it is now part of 'jaxws-rt'